jans-auth-server/common/src/test/java/io/jans/as/common/service/common/UserServiceImplTest.java

```diff
@@ -0,0 +1,30 @@
+package io.jans.as.common.service.common;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserServiceImplTest extends UserService {
```

Comment:
`TestUserService`, when we put suffix `*Test` it indicates classes with executable tests while it is not the case here.

Reply:
OK, it was fixed.